### DISCUSSION TITLE
Crime Analytica Architecture

### DISCUSSION THREAD

**Discussion by GarretTomlin** [Crime-Analytica/CrimeStop-Analytics#20] [Architecture] [answered]
![Crime Analytica drawio (1)](https://github.com/Crime-Analytica/CrimeStop-Analytics/assets/65048014/d96bdd9a-5edc-4572-9c59-9a97c98c0977)

**Comment by GarretTomlin** [comment]
@strique18 @100010We

**Comment by 100010We** [comment, marked as answer]
so we gonna have to add a feature where, when a person wants to report an alleged crime they can remain anonymous . If you haven't thought of that already that is

**Reply by GarretTomlin** [reply]
Yes we already have those features implemented the main node js API Is basically finished with a plethora of features that you can see here 

https://github.com/Crime-Analytica/CrimeStop-Analytics/tree/staging/backend/src/controllers

**Reply by 100010We** [reply]
Aiit, gonna check it now

**Comment by strique18** [comment]
What is the voice server?

**Reply by GarretTomlin** [reply]
using C++ under the hood https://mediasoup.org/
